deploy/kubernetes/controller.yaml

```diff
@@ -197,6 +198,19 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+          ports:
+            - name: healthz
+              containerPort: 9808
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 10
+            timeoutSeconds: 3
+            periodSeconds: 2
+            failureThreshold: 1
```

Comment:
`failureThreshold: 1` might be too aggressive, in their example they actually have 2:
```
failureThreshold: 5
...
failureThreshold: 1
```

Reply:
Looks like the test failures could have been caused because the the node driver restarted:
```
Logging pods the kubelet thinks is on node ip-172-20-50-17.us-west-2.compute.internal
Feb 26 02:54:34.076: INFO: kube-proxy-ip-172-20-50-17.us-west-2.compute.internal started at <nil> (0+0 container statuses recorded)
Feb 26 02:54:34.076: INFO: ebs-csi-node-xn2c8 started at 2019-02-26 02:33:06 +0000 UTC (0+3 container statuses recorded)
Feb 26 02:54:34.076: INFO: 	Container ebs-plugin ready: true, restart count 6
```

All 4 failures were on the volumes that take a long time to format and even if the formatting is attempted a second time, the 15 min timeout is not enough time.

Reply:
Notice the driver is keep being restarted even without any traffic due to probe timeout. Update the `periodSeconds` to 10s for less frequent probe, `failureThreshold` to 5 for higher failure tolerance